Lib/site.py

```diff
@@ -286,14 +295,16 @@ def joinuser(*args):
 def _get_path(userbase):
     version = sys.version_info
 
+    implementation = _get_impllibdir()
+    implementation_lower = implementation.lower()
     if os.name == 'nt':
         ver_nodot = sys.winver.replace('.', '')
-        return f'{userbase}\\Python{ver_nodot}\\site-packages'
+        return f'{userbase}\\{implementation}{ver_nodot}\\site-packages'
 
     if sys.platform == 'darwin' and sys._framework:
-        return f'{userbase}/lib/python/site-packages'
+        return f'{userbase}/lib/{implementation_lower}/site-packages'
 
-    return f'{userbase}/lib/python{version[0]}.{version[1]}/site-packages'
+    return f'{userbase}/lib/{implementation_lower}{version[0]}.{version[1]}/site-packages'
```

Comment:
Looking at this part of the code, it feels like the distinction is not impllibdir / implibdir_lower but impllibdir_windows and impllibdir_unix (or posix).

Reply:
Adopted a single `impllibdir` by passing `os.name` to `_get_impllibdir()`